packages/glibc.rb

```diff
@@ -7,28 +7,30 @@ class Glibc < Package
   compatibility 'all'
 
   depends_on 'gawk' => :build
-  depends_on 'libgd' => :build
   depends_on 'libidn2' => :build
   depends_on 'texinfo' => :build
   depends_on 'hashpipe' => :build
 
-  # enable-obsolete-rpc should NOT be included as a build option as
-  # this functionality is provided by libtirpc
+  no_env_options
+  conflicts_ok
 
-  case LIBC_VERSION
-  when '2.23'
-    version LIBC_VERSION
+  @LIBC_VERSION = LIBC_VERSION
```

Comment:
Why is this necessary?

Reply:
This will let you set the instance variable at the top so you can do cross-glibc-version builds, say you are on glibc 2.27 and want to build a glibc for 2.33, but you are not on a glibc 2.33 system.